examples/tv-app/android/java/MessagesManager.cpp

```diff
@@ -186,7 +186,7 @@ CHIP_ERROR MessagesManager::HandleGetMessages(AttributeValueEncoder & aEncoder)
             jint jduration         = env->GetIntField(messageObject, durationField);
             if (jduration >= 0)
             {
-                message.duration = DataModel::Nullable<uint16_t>(static_cast<uint16_t>(jduration));
+                message.duration = DataModel::Nullable<uint16_t>(static_cast<uint64_t>(jduration));
```

Comment:
other lines to change in this file due to data type change:

Line 77:
env->GetMethodID(managerClass, "presentMessages", "(Ljava/lang/String;IIJJLjava/lang/String;Ljava/util/HashMap;)Z");

Line 185:
jfieldID durationField = env->GetFieldID(messageClass, "duration", "J");
jlong jduration = env->GetLongField(messageObject, durationField);
if (jduration >= 0)
{
message.duration = DataModel::Nullable<uint64_t>(static_cast<uint64_t>(jduration));
}

Line 340:
jlong jduration = -1;
if (!duration.IsNull())
{
jduration = static_cast(duration.Value());
}

Reply:
Done.